java/src/main/java/org/rocksdb/HistogramType.java

```diff
@@ -9,24 +9,35 @@
   DB_GET(0),
   DB_WRITE(1),
   COMPACTION_TIME(2),
-  TABLE_SYNC_MICROS(3),
-  COMPACTION_OUTFILE_SYNC_MICROS(4),
-  WAL_FILE_SYNC_MICROS(5),
-  MANIFEST_FILE_SYNC_MICROS(6),
+  SUBCOMPACTION_SETUP_TIME(3),
+  TABLE_SYNC_MICROS(4),
+  COMPACTION_OUTFILE_SYNC_MICROS(5),
+  WAL_FILE_SYNC_MICROS(6),
+  MANIFEST_FILE_SYNC_MICROS(7),
```

Comment:
this has been changed from 6 to 7, do we need to change the counter part in C++ code as well?